src/ui.form-date-picker/config.js

```diff
@@ -1,9 +1,9 @@
 export default /*tw*/ {
   wrapper: "relative",
   input: "{UInput}",
-  inputFocused: {
+  activeInput: {
     component: "{UInput}",
```

Comment:
We don't use this as extending key because we need to reach to inner key of component